Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -781,6 +783,21 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.24,
 )
 
+class UpgradeWorkflow_trackingLowPUat0T(UpgradeWorkflowTracking,UpgradeWorkflow_0T):
+    def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Reco' in step and stepDict[step][k]['--era']=='Run3':
+            stepDict[stepName][k] = merge([{'--era': 'Run3_trackingLowPU'}, stepDict[step][k]])
+    def condition_(self, fragment, stepList, key, hasHarvest):
+        return ((fragment=="TTbar_14TeV" or fragment=="MinBias_14TeV") and ('2021' in key))
+upgradeWFs['trackingLowPUat0T'] = UpgradeWorkflow_trackingLowPUat0T(
+    steps = [
+        'Reco',
+    ],
+    PU = [],
+    suffix = '_trackingLowPUat0T',
+    offset = 0.424,
```

Comment:
Once you would like to specify new offset, please also document it in README (https://github.com/cms-sw/cmssw/blob/e57a6313443f3a7c6f8a7d053a92c007fc7e7cc3/Configuration/PyReleaseValidation/README.md). I understood that you are trying to combine 0.4 and 0.24 offsets together but this is a bit confusing. Is 0.24 related with 0.2 at all? maybe 0.41 is just good and move this class to below 0.4? We should try to avoid so many digits of the offset which already appeared. It becomes more confusing now :(

Reply:
Hi @chayanit, this PR is still in draft mode (not yet open for review)
Trying to get your point, nevertheless.
I think there are already several workflows in the current matrix with 4 offset digits. What makes them less confusing?